torchaudio/functional.py

```diff
@@ -652,3 +652,51 @@ def lowpass_biquad(waveform, sample_rate, cutoff_freq, Q=0.707):
     a1 = -2 * math.cos(w0)
     a2 = 1 - alpha
     return biquad(waveform, b0, b1, b2, a0, a1, a2)
+
+
+def compute_deltas(specgram, win_length=5):
+    # type: (Tensor, int) -> Tensor
+    r"""Compute delta coefficients of a spectogram:
```

Comment:
Nit: This technically isn't limited to spectrograms right? I mean, we could just talk about regular Tensors as inputs.

Also, the edge behavior might be something the user could want to change in the future. Some people rather not replicate the deltas, but just drop those frames altogether.

Reply:
> Nit: This technically isn't limited to spectrograms right? I mean, we could just talk about regular Tensors as inputs.

fair enough, it's commonly used on spectrogram, but i could replace this by `waveform`, since there is a time-component, though it is not limited to `waveform`. the list of [standardized words](https://github.com/pytorch/audio/blob/master/README.md) might not be useful here? :)

> Also, the edge behavior might be something the user could want to change in the future. Some people rather not replicate the deltas, but just drop those frames altogether.

This is the behavior that kaldi does, and also does not make configurable :) I'll add the option to change that, by simply passing along the mode option for `pad`, and default to 'replicate'.